assets/css/admin/style.scss

```diff
@@ -99,6 +99,30 @@ body.post-type-download #screen-meta {
 	margin: 0 0 0 20px;
 }
 
+.edd-header-page-title-wrap {
+	margin-top: 7px;
+	font-size: 22px;
+	line-height: 28px;
+	margin-right: auto;
+	padding-left: 7px;
+}
+
+.edd-header-separator {
+	margin-top: -2px;
+	opacity: 0.25;
+}
+
+.edd-header-page-title {
+	font-weight: 400;
+	font-size: 22px;
+	line-height: 28px;
```

Comment:
I'm not sure the line-height is necessary here....also what would you think of sizing the headings in `em` instead of `px`? `em` is what WP Core uses and it would be nice to start moving that way as we can.

Reply:
Without the line-height we end up with this:
<img width="567" alt="Screen Shot 2022-06-23 at 10 13 43 AM" src="https://user-images.githubusercontent.com/1390124/175356549-bcc66b49-743c-4159-8558-5a2a3a8440e4.png">

I'll adjust to using `em`